libs/networking/Http.ml

```diff
@@ -1,23 +1,12 @@
-open Http_lwt_client
-
-(* The http-lwt-client package offers:
-   - HTTP/1 and HTTP/2 support (using http/af and h2)
-   - TLS support (using OCaml-TLS)
-   - IPv4 and IPv6 connection establishment (via happy-eyeballs)
-*)
-
-(* The HTTP lwt client library uses happy_eyeballs as the underlying
-   layer for establishing connections. It uses DNS, and comes with a small DNS
-   cache. We use a single happy_eyeballs instance to reuse the cache present.
-*)
-let happy_eyeballs = Happy_eyeballs_lwt.create ()
-
-(* TODO: extend to allow to curl with JSON as answer *)
-let get ?headers url =
+open Piaf
```

Comment:
This will be in conflict with @ajbt200128 who is also modifying this file to use another Http library ...
We need to figure things out here and use just one if possible

Reply:
But yes we can move to libs/networking/

Reply:
We can also have Http_helpers1.ml and Http_helpers2.ml for now, where one use cohttp and ther other piaf
(or maybe http_helpers_piaf.ml and http_helpers_cohttp.ml)

Reply:
Also please keep as Http_helpers.ml, so it's clearer it's just some wrappers over another
official http library, otherwise it looks like we use an official Http robust library.

Reply:
We're switching to only Cohttp + moving networking to a libs + changing `Http.*` to `Http_helpers.*`